test/unit/test_Rewriting.cc

```diff
@@ -1,18 +1,22 @@
-//
-// Created by Martin Blicha on 17.05.20.
-//
+/*
+ * Copyright (c) 2020-2024 Martin Blicha <martin.blicha@gmail.com>
+ *
+ *  SPDX-License-Identifier: MIT
+ *
+ */
 
 #include <gtest/gtest.h>
-#include <ArithLogic.h>
-#include <BoolRewriting.h>
-#include <Logic.h>
-#include <ArithmeticEqualityRewriter.h>
-#include <DivModRewriter.h>
-#include <MainSolver.h>
-#include <DistinctRewriter.h>
+
+#include "ArithLogic.h"
+#include "BoolRewriting.h"
+#include "Logic.h"
+#include "ArithmeticEqualityRewriter.h"
```

Comment:
Why this is not included in `Rewritings.h`?

Reply:
I meant why wrapper functions on `ArithmeticEqualityRewriter` are not included

Reply:
I guess because I did not need that for what I wanted to achieve :)

But I guess we should write wrappers for these as well.